babel.config.js

```diff
@@ -0,0 +1,25 @@
+module.exports = api => {
```

Comment:
I assume this callback format for `babel.config.js` is new in v7.  I'm sure there are lots of cool things a person could do with this.